benchmark/msm/arkworks/Cargo.toml

```diff
@@ -18,6 +18,7 @@ publish = false
 
 [dependencies]
 ark-bn254 = "0.4.0"
+ark-ff = { version = "^0.4.0", features = ["asm"] }
```

Comment:
Can you make this a separate commit?

Reply:
Actually this change is a part of this commit! Since `infinity` is removed and this commit needs a trait from `ark-ff` to call `is_zero` method.